README.md

```diff
@@ -1,38 +1,106 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml)
 
-v8
+# v8
 
 A port of Google V8 JavaScript engine to z/OS Open Tools project
 
-# Applying the patches
+## Workflow 
+
+### Applying the patches
 Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go/+/master/cipd/) is not yet ported on z/OS, the following steps are
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
+This workflow requires two platforms.  One is USS which is the target platform.  The second is a generic linux
+AMD64 platform to get files normally acquired via `cipd`
+
+
+#### Linux AMD64 plaform setup
```

Comment:
Replace "USS" by "z/OS® UNIX System Services (USS for short)".
Also better not to specify "Linux AMD64", other user may not have it; I suggest replacing it by "non-USS".